pyrobuf/parse_proto.py

```diff
@@ -100,6 +100,7 @@ def __init__(self):
     def tokenize(self, s):
         pos = 0
         line = 1
+        _P = self.__class__
```

Comment:
Can we make this more descriptive? Also, why not just use `self`?

Reply:
Fixed :+1: